src/main/java/com/blogify/blogapi/model/User.java

```diff
@@ -39,30 +39,7 @@ public class User implements Serializable {
   @Getter(AccessLevel.NONE)
```

Comment:
why `AccessLevel.None` ?

Reply:
@NyH1409 I don't know exactly why this. 
@heiherilala can you help me to explain what this really means? Thanks

Reply:
it's for delete methode getter in the creationDatetimen. i want to dele setter heare but misklic. because creationDatetime is create automaticly in ccreation un can't be modify after
it's good with setter(... , but mybe, i'll delete it to be easier